app.R

```diff
@@ -1,93 +1,131 @@
 library(shiny)
-library(shinyWidgets)
+library(bslib)
 library(DT)
-library(ggplot2)
+library(dplyr)
```

Comment:
The package `shinyWidgets` is needed for `sliderTextInput` to work. 
`ggplot2` is also necessary or else we'll get an error saying `Error in aes: could not find function "aes"`.
I think we don't actually need to load `dplyr`, although I'm not quite sure why...